### PR TITLE
AmazonAWS.com.xml: Remove failing test

### PR DESCRIPTION
**Pull request by jayvdb**
Hostname is now in Law_Business_Research_CDN.xml

Fixes https://github.com/EFForg/https-everywhere/issues/18918

### CODE REVIEW

**Comment by jayvdb** [comment]
I assume is it safe to also remove the exclusion from this rule.

The other approach would be to remove the target from Law_Business_Research_CDN.xml .  For parsing that is a better approach, to avoid domains being split across multiple rulesets, but there are lots of other domains split across multiple rulesets so I guess that goal isnt achievable.

**Comment by cschanaj** [comment]
> The other approach would be to remove the target from Law_Business_Research_CDN.xml 

As you might note that `Law_Business_Research_CDN.xml` is not updated since 2014. There is some recent efforts to avoid merging multiple domain targets in the same ruleset. So I would recommend you to do it the other way.  

Besides, the rewrites in `Law_Business_Research_CDN.xml` will fail in September 2020 (#17912). It would be great if you could remove it in the PR as well.

File: src/chrome/content/rules/AmazonAWS.com.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <!--
 	For other Amazon coverage, see Amazon.xml.
 
+        For s3.lbrcdn.net.s3-external-3.amazonaws.com, see Law_Business_Research_CDN.xml
```
Comment (cschanaj) [inline src/chrome/content/rules/AmazonAWS.com.xml]: would you please remove this comment as well?

**Comment by cschanaj** [comment]
Thanks!

**Pull request merged**